RoslynPluginGenerator/DiagnosticAssemblyScanner.cs

```diff
@@ -90,7 +90,7 @@ private IEnumerable<DiagnosticAnalyzer> InstantiateDiagnosticsFromAssembly(strin
                     }
                     else
                     {
-                        this.logger.LogWarning(UIResources.Scanner_NoAnalyzers, analyzerAssembly.ToString());
+                        this.logger.LogInfo(UIResources.Scanner_NoAnalyzers, analyzerAssembly.ToString());
```

Comment:
Miscellaneous change: warning for every assembly that does not have analyzers is to noisy and produces warnings for valid packages that have helper assemblies e.g. SonarLint. Made this an info message instead. There will be one warning at the package level if none of the assemblies have analyzers.